server/src/services/LeadurServices.js

```diff
@@ -1,4 +1,6 @@
 const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const EmailServices = require('./EmailServices');
 
 const LeadurModel = require('../models/LeadurModel');
 const newUserSubscription = require('../subscribers/newUserSubscription');
```

Comment:
Get rid of the unused imports.
```suggestion
const LeadurModel = require('../models/LeadurModel');
```

Remove unused imports.